python/tutorials/gluon/02-layouts.py

```diff
@@ -0,0 +1,886 @@
+"""
+Tensor Layouts
+==============
+
+Tensors in Gluon require layouts. Layouts specify how the elements of the tensor
+are distributed among the threads a thread block. Tensors are distributed with
+respect to the hierarchy of the GPU beginning with thread blocks, then warps,
+lanes, and finally individual registers in each lane.
+
+Tensors are evenly distributed across theads, meaning that all threads own the
+same number of elements. Because Triton requires that all tile dimensions are
+powers of 2, this means that the number of elements per thread is a power of 2.
+
+A layout, in general, defines a mapping from a logical tensor index to the warp,
+lane, and register the element lives in. `BlockedLayout` is the most common
```

Comment:
It's in the opposite direction tho because of broadcasting.
```suggestion
A layout, in general, defines a mapping stating the element owned by a given
register, lane, and warp. `BlockedLayout` is the most common
```